python_hooks/forbidden_imports.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import ast
+import sys
+
+
+def check_imports(filename, forbidden_classes):
+    with open(filename) as file:
+        tree = ast.parse(file.read(), filename=filename)
+
+    for node in tree.body:
+        if isinstance(node, ast.Import) | isinstance(node, ast.ImportFrom):
+            for alias in node.names:
+                if alias.name in forbidden_classes:
+                    print(f"Flagged import of {alias.name} in {filename}")
+                    return 1  # Indicates failure
+    return 0  # Indicates success
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Check for specific imports')
+    parser.add_argument('--forbidden_classes', nargs='+', help='Names of the classes to check in imports')
+    parser.add_argument('file_list', nargs='+', help='List of files to check')  # provided by the pre-commit call
+    args = parser.parse_args()
+
+    classes_to_check = args.forbidden_classes
+    if not classes_to_check:
+        raise ValueError("No classes to check provided. add `args: ['--forbidden_classes', 'foo', 'bar', '--']` to the pre-commit config")
```

Comment:
I believe this check is handled by argparse, since forbidden-classes is a required arg.  Will this code ever execute?

Reply:
Sorry, I thought that flag was required, but it isn't.  You can disregard this.  😄

Reply:
yea, i commented out the `args` in the local testing and got: 
![image](https://github.com/tatari-tv/pre-commit-hooks/assets/12041723/1b302243-b092-4b60-a5bd-5b1e3ac5bb82)